cli/src/main/kotlin/commands/AdvisorCommand.kt

```diff
@@ -95,6 +96,7 @@ class AdvisorCommand : CliktCommand(name = "advise", help = "Run vulnerability d
     private fun configureAdvisor(advisorConfiguration: AdvisorConfiguration?): Advisor {
         val config = when (advisorConfiguration) {
             is NexusIqConfiguration -> advisorConfiguration
+            is VulnerableCodeConfiguration -> advisorConfiguration
```

Comment:
We might want to think about removing a default selection for the advisor.
IMO there is no sensible default for this choice, as both implementations require additional infrastructure.
Any opinions on that?

Reply:
I agree that we should not set a default advisor, but we actually don't do that here. We only fail (in line 100) if no advisor is configured, and that makes sense to do when running the `advise` sub-command. So, we should be all good, or?

Reply:
Yes we're good. And this change should also be done in a separate PR.

Reply:
Hmm, by "we're good" I actually meant "there is nothing to do" (not now, nor in a future PR). Or?

Reply:
Further up in this class a default is set:
https://github.com/oss-review-toolkit/ort/blob/2fad7307fdc28bf95e6508177d099960cff5ddf1/cli/src/main/kotlin/commands/AdvisorCommand.kt#L83-L89

Reply:
Ah, thanks, that we should fix in an upcoming PR indeed.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/3567.